src/Microsoft.Kusto.ServiceLayer/LanguageServices/ConnectedBindingQueue.cs

```diff
@@ -23,21 +18,11 @@ namespace Microsoft.Kusto.ServiceLayer.LanguageServices
     public class ConnectedBindingQueue : BindingQueue<ConnectedBindingContext>, IConnectedBindingQueue
     {
         internal const int DefaultBindingTimeout = 500;
-        private readonly ISqlConnectionOpener _connectionOpener;
         private readonly IDataSourceFactory _dataSourceFactory;
 
-        /// <summary>
-        /// Gets the current settings
-        /// </summary>
-        private SqlToolsSettings CurrentSettings
-        {
-            get { return WorkspaceService<SqlToolsSettings>.Instance.CurrentSettings; }
-        }
-
         [ImportingConstructor]
-        public ConnectedBindingQueue(ISqlConnectionOpener sqlConnectionOpener, IDataSourceFactory dataSourceFactory)
```

Comment:
We don't need a SqlConnection therefore we don't need a SqlConnectionOpener. Deleted ISqlConnectionOpener and SqlConnectionOpener